packages/atomic/tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "lib": ["dom", "es2020"],
     "moduleResolution": "node",
     "module": "esnext",
-    "target": "es2020",
+    "target": "es2019",
```

Comment:
If possible, I think it would be nice to add comments to the `tsconfig` file (and some other config files) when making changes like this.

I'm worried that later on we may find a reason why we want to upgrade the target and forget that we downgraded it and why we did so.